BlitzGateway 5.3.0

will-moore opened the pull request:
Tidy and reorganise BlitzGateway documentation page.
Didn't need to add any (much) different for 5.3.0, but I've improved the readability of the page by re-ordering some sections and removing some little bits. Fixing odd bits like "see above" at the start of the page etc.

Just needs a read-through to check for any similar glitches.

https://trello.com/c/PEvlrmHX/36-document-breaking-api-changes-for-devs-add-new-examples

will-moore committed:
Tidy and reorganise BlitzGateway documentation page

simleo commented on omero/developers/PythonBlitzGateway.txt:
```diff
@@ -1,19 +1,70 @@
+The Blitz Gateway is a Python client-side library that facilitates working
+with the OMERO API, handling connection to the server, loading of data objects
+and providing convenience methods to access the data. It was originally
+designed as part of the |OmeroWeb| framework, to provide connection and data
```
Remove "framework" (`|OmeroWeb|` already renders as "OMERO.web framework")

dominikl replied:
Same in line 139 (not introduced by the PR but could be fixed on that occasion, too).

simleo commented on omero/developers/PythonBlitzGateway.txt:
```diff
@@ -1,19 +1,70 @@
+The BlitzGateway class (see :pythondoc:`API of development code <omero/omero.gateway.html#omero.gateway._BlitzGateway>`)
+is a wrapper for the OMERO client and session objects. It provides
+various methods for connecting to the OMERO server, querying the status
+or context of the current connection and as a starting point for
```
Remove "as"

will-moore committed:
Updated OMERO.web landing page

will-moore committed:
Merge remote-tracking branch 'origin/develop' into BlitzGateway_5.3.0

will-moore committed:
Add OMERO.web and Python changes to whatsnew.txt

will-moore committed:
Remove 'framework' and 'as' as suggested in PR #1661

will-moore committed:
travis fix

jburel commented:
``/home/travis/build/openmicroscopy/ome-documentation/omero/developers/whatsnew.txt:16: WARNING: Inline interpreted text or phrase reference start-string without end-string.``

will-moore committed:
Adding ?maps and histograms to WebGateway.txt

will-moore committed:
Fix WARNING on whatsnew line 16

will-moore committed:
Add note about app config to CreateApp.txt

will-moore commented:
Added to What's New https://www.openmicroscopy.org/site/support/omero5.3-staging/developers/whatsnew.html 

Also updated https://www.openmicroscopy.org/site/support/omero5.3-staging/developers/Web.html
and https://www.openmicroscopy.org/site/support/omero5.3-staging/developers/Web/WebGateway.html 

Added note about App Config to https://www.openmicroscopy.org/site/support/omero5.3-staging/developers/Web/CreateApp.html

simleo commented on omero/developers/Web/CreateApp.txt:
```diff
@@ -164,6 +164,17 @@ Go to `http://localhost:4080/ <http://localhost:4080/>`_\<appname>/
+New in OMERO 5.3.0, we support the option of configuring your OMERO.web app with a
+name and label.
+See Django `Configuring Applications <https://docs.djangoproject.com/en/1.8/ref/applications/#configuring-applications>`_.
+This allows the URL to an app to be different from it's name.
```
"it's" -> "its"

will-moore committed:
Fix line 16 again

will-moore committed:
Fix it's -> its

hflynn commented on omero/developers/whatsnew.txt:
```diff
@@ -2,12 +2,18 @@ What's new for OMERO 5.3 for developers
 
+-  New ``api`` app to provide a new JSON API. See :doc:`/developers/json-api`
+-  Login functionality moved to Class-based LoginView in webgateway
+-  "Open with" config for opening apps from webclient. See :doc:`/developers/Web/LinkingFromWebclient'
```
You need a backtick at the end of this `:doc:` ref

will-moore committed:
Fix link in whatsnew.txt

will-moore committed:
Remove changes to whatsnew.txt

will-moore commented:
This is failing because it references json_api from #1627, so I'll just merge this branch into that one so everything is in a single PR.
